packages/astro/src/cli/add/index.ts

```diff
@@ -83,20 +83,16 @@ export default async function seed() {
 `,
 	CLOUDFLARE_WRANGLER_CONFIG: (name: string) => `\
 {
-	"main": "dist/_worker.js/index.js",
+	"compatibility_date": "2025-05-21",
 	"name": ${JSON.stringify(name)},
-  "compatibility_date": ${JSON.stringify(new Date().toISOString().slice(0, 10))},
-  "compatibility_flags": [
-    "nodejs_compat",
-    "global_fetch_strictly_public"
```

Comment:
It would be better to leave this one, as it's more intuitive behaviour when using workers for SSR rather than in front of another origin